server/lib/system/system.getNetworkMode.js

```diff
@@ -0,0 +1,33 @@
+const get = require('get-value');
+const { PlatformNotCompatible } = require('../../utils/coreErrors');
+const getConfig = require('../../utils/getConfig');
+
+const config = getConfig();
+
+/**
+ * @description Get used network by Gladys on Docker.
+ * @returns {Promise} Resolve Gladys used network.
+ * @example
+ * const containerNetwork = await this.getNetworkMode();
+ */
+async function getNetworkMode() {
+  if (!this.dockerode) {
+    throw new PlatformNotCompatible('SYSTEM_NOT_RUNNING_DOCKER');
+  }
+
+  if (!this.networkMode) {
+    // Get Gladys network
+    const containers = await this.getContainers({
+      all: false,
+    });
+    const gladysContainer = containers.find((c) => c.image.startsWith(config.dockerImage));
+    const gladysContainerData = await this.inspectContainer(gladysContainer.id);
+    this.networkMode = get(gladysContainerData, 'HostConfig.NetworkMode');
```

Comment:
I think this return the driver of the network and not the name.

From my test the all if fine but:

* on another bridged network ( like screenshot )
![image](https://user-images.githubusercontent.com/1528694/81456476-b1a73700-9192-11ea-81c3-b571228f08b1.png)
* the container is created in "bridge" network not "srv_default"

 I'm sorry, I hope my earlier comments didn't confuse you.

Reply:
Just did it, image is available.
Please try again.

Reply:
Error when I click 'Install broker'

`2020-05-09T11:20:41+0200 <error> installContainer.js:39 (MqttHandler.installContainer) MQTT broker failed to install as Docker container: TypeError: this.dockerode.getNetworks is not a function`

Typo ? must be getNetwork ( according to dockerode README )

Reply:
Oups... I will use `listNetworks` instead as I don't have the networkId. Build is in progress... another hour...

Reply:
Or 3 minutes for amd64 image :)

Reply:
Unfortunately eclipse-mosquitto is not created on the same network, you must use the network name not the network driver or type. ( Like I said in my previous comment )